test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -41,6 +42,8 @@ void test_random_iterator(const RandomIt& it) {
         (void) typename RandomIt::iterator_category{};
     }
 
+    EXPECT_TRUE(::std::is_default_constructible<RandomIt>::value, "iterator is not default constructible");
```

Comment:
I don't understand why we don't use `static_assert` for constant expression.

I suppose we also would like to call it at run-time, don't we?

Reply:
I believe that we don't need to call it at run-time, so yes, `static_assert` is better way to check constructability